_bibliography/published.bib

```diff
@@ -1,3 +1,26 @@
+@Article {Hinsen:2020,
```

Comment:
There is already an entry `Hinsen:2020` in `published.bib`. Do we have a naming convention for this situation? Pinging @rougier.